services/metadata_service/server.py

```diff
@@ -34,16 +34,16 @@ def app(loop=None, db_conf: DBConfiguration = None, middlewares=None):
     ArtificatsApi(app)
     AuthApi(app)
 
-    if len(PATH_PREFIX) > 0:
-        _app.add_subapp(PATH_PREFIX, app)
+    if path_prefix:
```

Comment:
would prefer explicitly checking for the default `if path_prefix is not None`

Reply:
Hmm I didn't want to treat "None" vs "" differently in any way here. Let me just change the function argument default to `""` for symmetry